themes/lotusdocs/layouts/partials/landing/hero.html

```diff
@@ -101,7 +101,7 @@ <h4 class="heading mt-0 mb-3">
                                         {{ .Content | safeHTML }}
                                     {{ else }}
                                         {{ with .Resize (printf "%dx%d webp q85 drawing Lanczos" .Width .Height) }}
-                                            <img src="{{ .RelPermalink }}" class="{{ if $boxShadow }}shadow-lg{{ else }}{{ end }} mx-auto d-block {{ if $rounded }}rounded-top-4{{ else }}{{ end }}" width="{{ .Width }}" height="{{ .Height }}" alt="{{ $alt }}">
+                                            <img src="{{ .RelPermalink }}" class="{{ if $boxShadow }}shadow-lg{{ else }}{{ end }} mx-auto d-block {{ if $rounded }}rounded-top-4{{ else }}{{ end }}" width=100% height=100% alt="{{ $alt }}">
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#width. % may not be expected here by some browsers. Use embedded css (`style="width: 100%; height: 100%;"`) instead?

Reply:
Fixed